challenges/type-unions/prompt.md

```diff
@@ -0,0 +1,70 @@
+## What Problem Type Unions Solve
+
+Type In a previous example for `type-aliases` we saw a problem.  We were writing code for a space shuttle and we wanted to make sure that we can't accidentally use the wrong units type.
```

Comment:
```suggestion
In a previous example for `type-aliases` we saw a problem.  We were writing code for a space shuttle and we wanted to make sure that we can't accidentally use the wrong units type.
```

typo?

Reply:
not sure if we should say in a previous problem but maybe it makes sense in the context of a track but this can be completed out of order and ad hoc

Reply:
conceptually speaking you can't complete this one, the way it's written, without having first learned of type aliases.  

concretely, the reason I did a short little recap is precisely so that you don't _really_ have to actually take the `type-aliases` one first.  my thinking is that the reality is for these first few that are REALLY foundational, the likelihood is that there is a little bit of an in-built order (only a little though! and only because of what's natural in terms of building up basic concepts!)

(fixed the typo in 7b97897c320a5bd616e2b579053e4c90621e62be)